src/screens/Notifications/NotificationScreen.tsx

```diff
@@ -0,0 +1,156 @@
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainCardStackNavigator>,
+    StackNavigationProp<MainCardStackNavigatorParams, Route.Notifications>
+  >;
+  wallets: any;
+}
+export class NotificationScreen extends Component<Props> {
+  onChangeEmailPress = () => {
+    Alert.alert('dsadasd');
+  };
+  onDeletePress = () => {};
+
+  onAddEmailPress = () => {
+    this.props.navigation.navigate(Route.AddEmail);
+  };
+
+  renderItem = (item: any) => {
+    return (
+      <TouchableOpacity style={styles.itemRow} onPress={() => this.goToWalletDetails(item.walletId)}>
+        <View style={styles.row}>
+          <Text style={styles.walletName}>{item.name}</Text>
+          <Image source={images.backArrow} style={styles.arrow} resizeMode="contain" />
+        </View>
+        <Text style={styles.caption}>{item.description}</Text>
+      </TouchableOpacity>
+    );
+  };
+
+  goToWalletDetails = (walletId: string) => {
+    this.props.navigation.navigate(Route.WalletDetails, { id: walletId });
+  };
+
+  renderFooter = () =>
+    this.props.wallets ? (
+      <>
+        <Button title={i18n.notifications.change} onPress={this.onChangeEmailPress} />
+        <FlatButton
+          containerStyle={styles.deleteButton}
+          buttonType={ButtonType.Warning}
+          title={i18n.notifications.delete}
+          onPress={this.onDeletePress}
+        />
+      </>
+    ) : (
+      <Button title={i18n.notifications.addEmail} onPress={this.onAddEmailPress} />
+    );
+
+  render() {
+    return (
+      <ScreenTemplate
+        header={<Header isBackArrow={true} title={i18n.settings.notifications} />}
+        noScroll
+        footer={this.renderFooter()}
+      >
+        {this.props.wallets ? (
+          <>
+            <Text style={styles.title}>{i18n.notifications.title}</Text>
+            <Text style={styles.description}>{i18n.notifications.description}</Text>
+            <View style={styles.amountInput}>
+              <Text style={styles.amount}>kamil2912@gmail.com</Text>
+            </View>
+            <Text style={styles.listTitle}>{i18n.notifications.yourSubscriptions}</Text>
+
+            <FlatList
+              data={this.props.wallets}
+              renderItem={item => this.renderItem(item.item)}
+              keyExtractor={item => item.id}
+            />
+          </>
+        ) : (
+          <View style={styles.noWalletsContainer}>
+            <Text style={typography.headline4}>{i18n.notifications.addYourAddress}</Text>
+            <Image source={images.bigBell} style={styles.bigBell} resizeMode="contain" />
+            <Text style={styles.noWalletsDescription}>{i18n.notifications.addYourAddressDescription}</Text>
+          </View>
+        )}
+      </ScreenTemplate>
+    );
+  }
+}
+
+// @ts-ignore
+NotificationScreen.defaultProps = {
+  wallets: undefined,
+  // wallets: [{ id: '1', name: 'sdsadasdasdas', description: 'dsadsadas' }],
```

Comment:
If it's for debugging and will be removed later on, I'd leave a TODO comment to not forget

Reply:
done